python/python/glide/constants.py

```diff
@@ -33,8 +33,29 @@
 TSingleNodeRoute = Union[RandomNode, SlotKeyRoute, SlotIdRoute, ByAddressRoute]
 # When specifying legacy path (path doesn't start with `$`), response will be T
 # Otherwise, (when specifying JSONPath), response will be List[Optional[T]].
+#
+# Explanation:
+# TJsonResponse is designed to handle scenarios where some paths may not contain valid values, especially with JSONPath targeting multiple paths.
+# In such cases, the response may include None values, represented as `Optional[T]` in the list.
+# This type provides flexibility for commands where a subset of the paths may return None.
+#
 # For more information, see: https://redis.io/docs/data-types/json/path/ .
 TJsonResponse = Union[T, List[Optional[T]]]
+
+# When specifying legacy path (path doesn't start with `$`), response will be T
+# Otherwise, (when specifying JSONPath), response will be List[T].
+# This type represents the response format for commands that apply to every path and every type in a JSON document.
+# It covers both singular and multiple paths, ensuring that the command returns valid results for each matched path without None values.
+#
+# Explanation:
+# TJsonUniversalResponse is considered "universal" because it applies to every matched path and
+# guarantees valid, non-null results across all paths, covering both singular and multiple paths.
+# This type is used for commands that return results from all matched paths, ensuring that each
+# path contains meaningful values without None entries (unless it's part of the commands response).
+# It is typically used in scenarios where each target is expected to yield a valid response. For commands that are valid for all target types.
+#
+# For more information, see: https://redis.io/docs/data-types/json/path/ .
```

Comment:
Can we refer to redis docs?

Reply:
Really tried to find this in valkey, will try again

Reply:
If no such doc, delete the link and/or leave a TODO